osgp/platform/osgp-adapter-ws-core/src/main/java/org/opensmartgridplatform/adapter/ws/core/application/mapping/DeviceConverterHelper.java

```diff
@@ -180,4 +181,12 @@ public XMLGregorianCalendar convertDateToXMLGregorianCalendar(final Date date) {
         return xmlCalendar;
     }
 
+    public XMLGregorianCalendar convertInstantToXMLGregorianCalendar(final Instant instant) {
+        if (instant == null) {
+            return null;
+        } else {
```

Comment:
Noting wrong with this and lately I have seen more people do this.
It just does not align with how I like to look at things and structure the code.
I like to consider the special case return value separate from the normal logic. With the else it feels like everything gets tied together in a bigger logical structure that does not need to be there.
Now this is a small method (which is good) and it does not matter that much, but there is also the level of indentation that you would not need for what comes in `else` now.
Just wanted to share my view, maybe even convince you, but feel free to ignore this if you like it as is.
The way `null` is handled here is way nicer anyway (in my opinion) than how it is handled in `convertDateToXMLGregorianCalendar`.

Reply:
Refactored code. (Removed the convert methods and use mapper instead)